src/commissaire/storage/__init__.py

```diff
@@ -29,7 +31,10 @@ class StoreHandlerBase(object):
     Base class for all StoreHandler classes.
     """
 
-    # Subclasses override this, if applicable.
+    #: Class level LOGGER
+    logger = logging.getLogger('StorageHandler')
```

Comment:
Maybe use the actual class name for the logger name, like you do for `ServiceManager`?  (or not, if you disagree)